mobility_pipeline/lib/overlap.py

```diff
@@ -0,0 +1,9 @@
+from shapely.geometry import Polygon
+
```

Comment:
Can you add a docstring here to summarize what this file is for? Docstrings are triple-quoted strings that aren't assigned to anything, and the format is described in the [contributing guide](https://github.com/codethechange/mobility_pipeline/blob/master/CONTRIBUTING.rst#code-style) (we use the Google style). For an example, see the [data interface file](https://github.com/codethechange/mobility_pipeline/blob/f007ec683b66a8ff6747225d7748fc2c498e73af/mobility_pipeline/data_interface.py#L1).